redis/tests/support/mod.rs

```diff
@@ -349,28 +349,7 @@ impl TestContext {
         Self::with_modules(&[], true)
     }
 
-    pub fn with_tls(tls_files: TlsFilePaths, mtls_enabled: bool) -> TestContext {
```

Comment:
why did you move this?

Reply:
I did not, I moved the common connection establishment code (now in use by 3 funcs) into connect_with_retries()

Reply:
git issue